oshi-core/src/main/java/oshi/software/os/linux/LinuxOperatingSystem.java

```diff
@@ -262,7 +262,7 @@ public int getProcessCount() {
 
     @Override
     public int getThreadId() {
-        return LinuxLibc.INSTANCE.gettid();
```

Comment:
One other rather simple fallback which should be cross-platform:  just read `/proc/thread-self` and return the value of `Pid` (which is the tid in the proc/pid/task tree)